samples/apps/copilot-chat-app/webapi/CopilotChat/Skills/ChatSkills/ChatSkill.cs

```diff
@@ -158,9 +159,26 @@ public async Task<string> ExtractChatHistoryAsync(SKContext context)
             var formattedMessage = chatMessage.ToFormattedString();
             var tokenCount = Utilities.TokenCount(formattedMessage);
 
-            // Plan object is not meaningful content in generating chat response, exclude it
-            if (remainingToken - tokenCount > 0 && !formattedMessage.Contains("proposedPlan\\\":", StringComparison.InvariantCultureIgnoreCase))
+            if (remainingToken - tokenCount > 0)
             {
+                // Plan object is not meaningful content in generating chat response, reduce to intent to save on tokens
+                if (formattedMessage.Contains("proposedPlan\":", StringComparison.InvariantCultureIgnoreCase))
```

Comment:
Before (at the old line 162), we were looking for proposedPlan followed by a backslash, followed by a double-quote.

Now we are looking for proposedPlan immediately followed by a double-quote, followed by a semi-colon.  Is this what you wanted to do?

Reply:
Yep the old regex was wrong, we were looking for an extra backslash